app/models/manageiq/providers/vmware/network_manager.rb

```diff
@@ -19,7 +19,6 @@ class ManageIQ::Providers::Vmware::NetworkManager < ManageIQ::Providers::Network
            :authentication_status_ok?,
            :authentications,
            :authentication_for_summary,
-           :zone,
```

Comment:
Was zone included here for a valid reason?  I'm curious why it was delegated in the first place.

Reply:
some places we use zone and other places we use zone_id

we tried to replicate the zone across from parent to child, but the autosave makes it a little tricky

Reply:
This is how we handled keeping zone "in sync" with the parent manager originally, but we found that (specifically in the per_ems_worker_mixin code) doing `ExtManagementSystem.where(:zone => my_zone)` these were not picked up.

We now have callbacks that set `child.zone_id = zone_id` when the cloud_manager is updated.